spacemacs/packages.el

```diff
@@ -3708,10 +3708,15 @@ one of `l' or `r'."
               (,(listify-key-sequence
                  (kbd dotspacemacs-emacs-leader-key)) . "Spacemacs root")))
       (nconc which-key-prefix-title-alist spacemacs/prefix-titles)
+      (pcase dotspacemacs-which-key-position
+        ('right (which-key-setup-side-window-right))
+        ('bottom (which-key-setup-side-window-bottom))
+        ('right-then-bottom (which-key-setup-side-window-right-bottom)))
```

Comment:
Why? Without using commas they should be equivalent, no?
On Wed, Sep 2, 2015 at 6:30 AM Markus Bertheau <notifications@github.com>
wrote:

> In spacemacs/packages.el
> <https://github.com/syl20bnr/spacemacs/pull/2812#discussion_r38517928>:
>
> > @@ -3708,10 +3708,15 @@ one of `l' or `r'."
> >                (,(listify-key-sequence
> >                   (kbd dotspacemacs-emacs-leader-key)) . "Spacemacs root")))
> >        (nconc which-key-prefix-title-alist spacemacs/prefix-titles)
> > +      (pcase dotspacemacs-which-key-position
> > +        ('right (which-key-setup-side-window-right))
> > +        ('bottom (which-key-setup-side-window-bottom))
> > +        ('right-then-bottom (which-key-setup-side-window-right-bottom)))
>
> This has to be
>
> (`right (...))
> (`bottom (...))
> ...
>
> Note the backquote instead of the straight single quote.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/2812/files#r38517928>.
>